plugins/InspectionGadgets/test/com/siyeh/igtest/controlflow/if_statement_with_identical_branches/IfStatementWithIdenticalBranches.java

```diff
@@ -6,21 +6,31 @@
 
 public class IfStatementWithIdenticalBranches {
 
-    void one() {
-        <warning descr="'if' statement with identical branches">if</warning> (true) {
+    void simpleCases(boolean isValid) {
+        if (isValid) {
 
         } else {
 
         }
-        <warning descr="'if' statement with identical branches">if</warning> (false) {
+        <weak_warning descr="'if' statement can be collapsed">if</weak_warning> (isValid) {
```

Comment:
Why changes in existing test data?

Reply:
The unit test was missing so probably the test cases were not maintained any longer. I needed to update them to make the tests work.